systemd/bootupd.service

```diff
@@ -19,7 +19,6 @@ ProtectHome=yes
 ReadOnlyPaths=/usr
 PrivateTmp=yes
 PrivateNetwork=yes
-ProtectClock=yes
```

Comment:
Probably something going wrong in the seccomp policy systemd makes.  It's...about time...we looked at that.